caffe2/utils/threadpool/pthreadpool.h

```diff
@@ -33,48 +33,6 @@ typedef void (*pthreadpool_function_4d_tiled_t)(
     size_t,
     size_t);
 
-#include <stdint.h> // for uint32_t
-
-typedef void (*pthreadpool_task_1d_t)(void*, size_t);
-typedef void (*pthreadpool_task_1d_tile_1d_t)(void*, size_t, size_t);
-typedef void (*pthreadpool_task_2d_t)(void*, size_t, size_t);
-typedef void (*pthreadpool_task_2d_tile_1d_t)(void*, size_t, size_t, size_t);
-typedef void (*pthreadpool_task_2d_tile_2d_t)(void*, size_t, size_t, size_t, size_t);
-typedef void (*pthreadpool_task_3d_tile_2d_t)(
-    void*,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t);
-typedef void (*pthreadpool_task_4d_tile_2d_t)(
-    void*,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t);
-typedef void (*pthreadpool_task_5d_tile_2d_t)(
-    void*,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t);
-typedef void (*pthreadpool_task_6d_tile_2d_t)(
-    void*,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t,
-    size_t);
```

Comment:
The extended pthreadpool API is no longer needed for C2's implementation, as was the case before.

Reply:
Because C2 was never using this API? It was mainly for xnnpack I think. Is that right?

Reply:
Yes, that's correct.  This new API is only used for XNNPACK, which is now using the third-party version.